marketplaceItems/appreciate/appResources/appData/appreciate_app.js

```diff
@@ -0,0 +1,947 @@
+        "x": 0.24,
+        "y": 0.24,
+        "z": 0.24
+    };
+    var INTENSITY_ENTITY_MIN_ANGULAR_VELOCITY = {
+        "x": -0.21,
+        "y": -0.21,
+        "z": -0.21
+    };
+    var INTENSITY_ENTITY_MAX_ANGULAR_VELOCITY = {
+        "x": 0.21,
+        "y": 0.21,
+        "z": 0.21
+    };
+    var intensityEntityColorMin = {
+        "red": 82,
+        "green": 196,
+        "blue": 145
+    };
+    var INTENSITY_ENTITY_COLOR_MAX_DEFAULT = {
+        "red": 5,
+        "green": 255,
+        "blue": 5
+    };
+    var intensityEntityColorMax = JSON.parse(Settings.getValue("appreciate/entityColor",
+        JSON.stringify(INTENSITY_ENTITY_COLOR_MAX_DEFAULT)));
+    var ANGVEL_ENTITY_MULTIPLY_FACTOR = 62;
+    var INTENSITY_ENTITY_PROPERTIES = {
+        "type": "Shape",
+        "shape": "Dodecahedron",
+        "dimensions": {
+            "x": 0,
+            "y": 0,
+            "z": 0
+        },
+        "angularVelocity": {
+            "x": 0,
+            "y": 0,
+            "z": 0
+        },
+        "angularDamping": 0,
+        "grab": {
+            "grabbable": false,
+            "equippableLeftRotation": {
+                "x": -0.0000152587890625,
+                "y": -0.0000152587890625,
+                "z": -0.0000152587890625,
+                "w": 1
+            },
+            "equippableRightRotation": {
+                "x": -0.0000152587890625,
+                "y": -0.0000152587890625,
+                "z": -0.0000152587890625,
+                "w": 1
+            }
+        },
+        "collisionless": true,
+        "ignoreForCollisions": true,
+        "queryAACube": {
+            "x": -0.17320507764816284,
+            "y": -0.17320507764816284,
+            "z": -0.17320507764816284,
+            "scale": 0.3464101552963257
+        },
+        "damping": 0,
+        "color": intensityEntityColorMin,
+        "clientOnly": false,
+        "avatarEntity": true,
+        "localEntity": false,
+        "faceCamera": false,
+        "isFacingAvatar": false
+    };
+    var INTENSITY_MATERIAL_ENTITY_PROPS = {
+        "name": "Intensity Entity Material",
+        "type": "Material",
+        "materialURL": "materialData"
+    };
+    var currentInitialAngularVelocity = {
+        "x": 0,
+        "y": 0,
+        "z": 0
+    };
+    function updateIntensityEntity() {
+        if (currentIntensity > 0) {
+            if (intensityEntity) {
+                intensityEntityColorMin.red = intensityEntityColorMax.red * 0.4;
+                intensityEntityColorMin.green = intensityEntityColorMax.green * 0.4;
+                intensityEntityColorMin.blue = intensityEntityColorMax.blue * 0.4;
+
+                var color = linearScaleColor(currentIntensity, intensityEntityColorMin, intensityEntityColorMax);
+
+                if (intensityMaterialEntity) {
+                    Entities.editEntity(intensityMaterialEntity, {
+                        materialData: JSON.stringify({"materialVersion": 1,
```

Comment:
can that materialVersion  go on the following line

Reply:
yes!